classes/api/entries.php

```diff
@@ -102,6 +118,50 @@ public function delete_item( WP_REST_Request $request ) {
 		}
 	}
 
+    /**
+     * Delete all entries of a form based on Form ID
+     *
+     * GET /cf-api/v2/entries/form-id/delete
+     *
+     * @since 1.7.0
+     *
+     * @param WP_REST_Request $request
+     *
+     * @return Caldera_Forms_API_Error|Caldera_Forms_API_Response
+     */
+    public function delete_entries( WP_REST_Request $request ) {
+        try{
+            $this->form_object_factory( $request[ 'form_id' ], $request );
+        }catch ( Exception $e ){
+            return Caldera_Forms_API_Response_Factory::error_form_not_found();
+        }
+
+        $entries = \calderawp\CalderaFormsQueries\CalderaFormsQueries()->selectByFormId( $request[ 'form_id' ], false );
```

Comment:
@New0 You should validate that form exists first.

Reply:
@Shelob9 Isn't what `$this->form_object_factory( $request[ 'form_id' ], $request );` does in try and returning catch when it doesn't find a form object associated to the form ID ?